projects/microphysics/argo/end_to_end.py

```diff
@@ -0,0 +1,222 @@
+"""Configuration class for submitting end to end experiments with argo
+"""
+from copy import deepcopy
+import yaml
+import subprocess
+import dataclasses
+import base64
+import os
+from fv3net.artifacts.resolve_url import resolve_url
+import pathlib
+import sys
+
+
+WORKFLOW_FILE = pathlib.Path(__file__).parent / "argo.yaml"
+
+
+def load_yaml(path):
+    return yaml.safe_load(pathlib.Path(path).read_bytes())
+
+
+def submit(job, labels, other_parameters=None):
+    other_parameters = other_parameters or {}
+    args = (
+        [
+            "argo",
+            "submit",
+            WORKFLOW_FILE.absolute().as_posix(),
+            "--entrypoint",
+            job.entrypoint,
+            "--name",
+            job.name,
+        ]
+        + _argo_parameters(job.parameters + other_parameters)
+        + _label_args(labels)
+    )
+    subprocess.check_call(args, stdout=subprocess.DEVNULL)
+
+
+def _run_job_q(job):
+    print(job)
+    return input("Y/n") != "n"
+
+
+def submit_jobs(jobs, experiment_name):
```

Comment:
Brief docstring mostly to delineate that experiment name is used for grouping by tag/group.  Could add that `jobs` is a `Sequence[JobType]` or whatever you settle on for the type into `submit_job`.